### PR TITLE
Copy translations update

### PR DESCRIPTION
**Pull request by aitorvs**
Task/Issue URL: https://app.asana.com/0/72649045549333/1199369362194383/f
Tech Design URL: 
CC: 

**Description**:
Update copy translations (see [task](https://app.asana.com/0/72649045549333/1199369362194383/f) for more details)

**Steps to test this PR**:
1. 

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: app/src/main/res/values-hr/strings.xml [hunk 1]
```diff
@@ -405,41 +407,41 @@
...
+    <string name="onboardingDaxText"><![CDATA[Internet može biti pomalo neugodan.<br/><br/> Ne brini! Privatno pretraživanje i pregledavanje lakše je nego što misliš.]]></string>
     <string name="onboardingLetsDoItButton">Učinimo to!</string>
-    <string name="daxIntroCtaText"><![CDATA[Zatim pokušajte posjetiti jedno od svojih omiljenih web-mjesta! <br/><br/>Blokirat ću tragače kako vas ne bi mogli špijunirati. Nadogradit ću i sigurnost vaše veze ako je moguće. 🔒]]></string>
-    <string name="daxEndCtaText"><![CDATA[<br/><br/>Shvatili ste ovo! Zapamtite: svaki put kada pregledavate sa mnom jeziva reklama gubi krila. 👍]]></string>
-    <string name="daxSerpCtaText">Vaše pretrage u DuckDuckGo anonimne su i nikad ne pohranjujem vašu povijest pretraga. Nikad. 🙌</string>
+    <string name="daxIntroCtaText"><![CDATA[Zatim pokušaj posjetiti jedno od svojih omiljenih web-mjesta! <br/><br/>Blokirat ću tragače kako te ne bi mogli špijunirati. Nadogradit ću i sigurnost tvoje veze ako je moguće. 🔒]]></string>
+    <string name="daxEndCtaText"><![CDATA[Možeš ti to!\n\nZapamti: svaki put kada pregledavaš sa mnom grozna reklama gubi krila. 👍]]></string>
```
Comment (cmonfortep) [inline app/src/main/res/values-hr/strings.xml:421]: `<br>` replaced by `\n` inside a `CDATA` block.

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -397,37 +399,37 @@
...
-    <string name="onboardingDaxText"><![CDATA[Internet peut être intrusif.<br/><br/>Cependant ne vous inquiétez pas ! Rechercher et naviguer de manière confidentielle est plus facile que vous ne le pensez.]]></string>
-    <string name="onboardingLetsDoItButton">Allons-y !</string>
-    <string name="daxIntroCtaText"><![CDATA[Maintenant, essayez de visiter un de vos sites préférés !<br/><br/>Je bloquerai les traqueurs afin qu\'ils ne puissent pas vous espionner. J\'améliorerai également la sécurité de votre connexion dans la mesure du possible. 🔒]]></string>
-    <string name="daxEndCtaText"><![CDATA[Bien joué !<br/><br/>Rappelez vous : chaque fois que vous naviguez avec moi, des publicités intrusives disparaissent. 👍]]></string>
-    <string name="daxSerpCtaText">Vos recherches effectuées sur DuckDuckGo sont confidentielles et je ne conserve jamais votre historique de recherche. Jamais. 🙌</string>
+    <string name="onboardingDaxText"><![CDATA[Internet peut être intrusif.\n\nMais ne vous inquiétez pas ! Rechercher et naviguer de manière confidentielle est plus facile que vous ne le pensez.]]></string>
```
Comment (cmonfortep) [inline app/src/main/res/values-fr/strings.xml:410]: `<br>` replaced by `\n` inside a `CDATA` block.

File: app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -397,37 +399,37 @@
...
+    <string name="onboardingLetsDoItButton">Allons-y !</string>
+    <string name="daxIntroCtaText"><![CDATA[Maintenant, essayez de visiter un de vos sites préférés !<br/><br/>Je bloquerai les traqueurs afin qu\'ils ne puissent pas vous espionner. J\'améliorerai également la sécurité de votre connexion dans la mesure du possible. 🔒]]></string>
+    <string name="daxEndCtaText"><![CDATA[Bien joué !\n\nSouvenez-vous : chaque fois que vous naviguez avec moi, des publicités intrusives disparaissent. 👍]]></string>
```
Comment (cmonfortep) [inline app/src/main/res/values-fr/strings.xml:413]: `<br>` replaced by `\n` inside a `CDATA` block.

File: app/src/main/res/values-lv/strings.xml [hunk 4]
```diff
@@ -360,90 +362,90 @@
...
     <string name="hideTipsButton">Paslēpt padomus uz visiem laikiem</string>
 
     <!-- New Onboarding Experience -->
     <string name="onboardingWelcomeTitle">"Laipni lūdzam DuckDuckGo!"</string>
-    <string name="onboardingDaxText"><![CDATA[Internets var būt diezgan biedējošs.<br/><br/>Bet neuztraucieties! Meklēt un pārlūkot privātā režīmā ir vieglāk, nekā jūs domājat.]]></string>
-    <string name="onboardingLetsDoItButton">Darīsim to!</string>
-    <string name="daxIntroCtaText"><![CDATA[Tagad pamēģiniet ieiet kādā no jūsu mīļākajām vietnēm! <br/><br/>Es nobloķēšu izsekotājus, lai viņi nevarētu jūs izspiegot. Ja iespējams, uzlabošu arī jūsu savienojuma drošību. 🔒]]></string>
-    <string name="daxEndCtaText"><![CDATA[Tas izdevās! <br/><br/>Atcerieties – katru reizi, kad jūs pārlūkojat ar mani, jocīgas reklāmas atkāpjas. 👍]]></string>
-    <string name="daxSerpCtaText">Jūsu DuckDuckGo meklējumi ir anonīmi, un es nekad neuzglabāju jūsu meklēšanas vēsturi. Nekad. 🙌</string>
+    <string name="onboardingDaxText"><![CDATA[Internets var būt diezgan “mežonīgs”.\n\nBet neuztraucies! Meklēt un pārlūkot privātā režīmā ir vieglāk, nekā tu domā.]]></string>
```
Comment (cmonfortep) [inline app/src/main/res/values-lv/strings.xml:414]: `<br>` replaced by `\n` inside a `CDATA` block.

File: app/src/main/res/values-nb/strings.xml [hunk 5]
```diff
@@ -399,28 +401,28 @@
...
+        <item quantity="other"><![CDATA[ prøvde å spore deg her. <br/><br/>Jeg blokkerte dem!<br/><br/> ☝️Du kan sjekke adresselinjen for å se hvem som prøver å spore deg når du besøker et nytt nettsted.️]]></item>
     </plurals>
-    <string name="daxNonSerpCtaText"><![CDATA[Når du trykker og blar, blokkerer jeg irriterende trackere. <br/><br/>Gå videre – fortsett å bla!]]></string>
-    <string name="daxMainNetworkCtaText"><![CDATA[Følg med! %1$s er et stort sporingsnettverk.<br/><br/> Trackerne deres lurer på omtrent %2$s av ledende nettsteder 😱 men ikke bekymre deg! <br/><br/>Jeg hindrer %3$s fra å se aktiviteten din på disse nettstedene.]]></string>
-    <string name="daxMainNetworkOwnedCtaText">Vennligst merk! Siden %1$s eier %2$s kan jeg ikke hindre dem fra å se aktiviteten din her.&lt;br/&gt;&lt;br/&gt;Men surf med meg, så kan jeg redusere informasjonen %3$s får om deg ved å blokkere sporingsanordningene deres på mange andre nettsteder.</string>
+    <string name="daxNonSerpCtaText"><![CDATA[Jeg blokkerer irriterende sporere mens du trykker og blar.\n\nDu kan bare fortsette å surfe!]]></string>
```
Comment (cmonfortep) [inline app/src/main/res/values-nb/strings.xml:423]: `<br>` replaced by `\n` inside a `CDATA` block.

File: app/src/main/res/values-nl/strings.xml [hunk 6]
```diff
@@ -404,30 +406,30 @@
...
 
     <!-- New Onboarding Experience -->
-    <string name="onboardingWelcomeTitle">"WELKOM BIJ DUCKDUCKGO!"</string>
-    <string name="onboardingDaxText"><![CDATA[Het internet kan best eng zijn.<br/><br/>Maak je geen zorgen! Privé zoeken en browsen is eenvoudiger dan u denkt.]]></string>
-    <string name="onboardingLetsDoItButton">Laten we het Doen!</string>
-    <string name="daxIntroCtaText"><![CDATA[Probeer vervolgens een van uw favoriete sites te bezoeken! <br/><br/>Ik blokkeer trackers zodat ze u niet kunnen bespioneren. Ik zal ook indien mogelijk de beveiliging van uw verbinding upgraden. 🔒]]></string>
-    <string name="daxEndCtaText"><![CDATA[U kunt het! <br/><br/>Denk eraan: elke keer als u met mij surft, verliest een enge advertentie zijn vleugels. 👍]]></string>
-    <string name="daxSerpCtaText">Uw DuckDuckGo-zoekopdrachten zijn anoniem en ik sla uw zoekgeschiedenis nooit op. Nooit. 🙌</string>
+    <string name="onboardingWelcomeTitle">"Welkom bij DuckDuckGo!"</string>
+    <string name="onboardingDaxText"><![CDATA[Internet kan best eng zijn.\n\nMaak je geen zorgen! Privé zoeken en browsen is eenvoudiger dan je denkt.]]></string>
```
Comment (cmonfortep) [inline app/src/main/res/values-nl/strings.xml:410]: `<br>` replaced by `\n` inside a `CDATA` block.

File: app/src/main/res/values-sv/strings.xml [hunk 7]
```diff
@@ -495,37 +497,37 @@
...
-    <string name="preciseLocationSystemDialogSubtitle">%1$s begär åtkomst till din enhetsplats. Du måste tillåta att appen skickar din enhetsplats innan %2$s kan be om åtkomst.</string>
+    <string name="preciseLocationSystemDialogTitle">Aktivera åtkomst till enhetens platsinf\bormation för DuckDuckGo Privacy Browser?</string>
+    <string name="preciseLocationSystemDialogSubtitle">%1$s begär åtkomst till din enhets platsinf\bormation. Du måste tillåta att appen skickar din enhets platsinf\bormation innan %2$s kan be om åtkomst.</string>
     <string name="preciseLocationSystemDialogAllow">Aktivera</string>
     <string name="preciseLocationSystemDialogDeny">Kanske senare</string>
     <string name="preciseLocationSystemDialogNeverAllow">Fråga inte igen för denna webbplats</string>
-    <string name="preciseLocationSiteDialogTitle">Bevilja åtkomst till enhetsplats för %1$s?</string>
-    <string name="preciseLocationSiteDialogSubtitle">Du kan gå till Inställningar för att hantera åtkomster till enhetsplats som du har beviljat för enskilda webbplatser.</string>
+    <string name="preciseLocationSiteDialogTitle">Bevilja åtkomst till enhetens platsinf\bormation för %1$s?</string>
```
Comment (cmonfortep) [inline app/src/main/res/values-sv/strings.xml:506]: That `\` is not going to work if it's not escaped.

File: app/src/main/res/values-sv/strings.xml [hunk 8]
```diff
@@ -495,37 +497,37 @@
     <string name="downloadManagerIncompatible">Nerladdningshanteraren är inte tillgänglig på denna enhet</string>
     <string name="enable">Aktivera</string>
 
-    <!--Fire button education-->
-    <string name="daxClearDataCtaText">Tryck här nedanför nu för att bränna din webbhistorik. Prova! 👇</string>
-
     <!-- Precise Location -->
-    <string name="preciseLocationSystemDialogTitle">Aktivera åtkomst till enhetsplats för DuckDuckGo Privacy Browser?</string>
-    <string name="preciseLocationSystemDialogSubtitle">%1$s begär åtkomst till din enhetsplats. Du måste tillåta att appen skickar din enhetsplats innan %2$s kan be om åtkomst.</string>
+    <string name="preciseLocationSystemDialogTitle">Aktivera åtkomst till enhetens platsinf\bormation för DuckDuckGo Privacy Browser?</string>
```
Comment (cmonfortep) [inline app/src/main/res/values-sv/strings.xml:501]: non escaped char here.

File: app/src/main/res/values-sv/strings.xml [hunk 9]
```diff
@@ -495,37 +497,37 @@
...
-    <string name="preciseLocationToggleText">Webbplatser kan be om åtkomst till min enhetsplats</string>
-    <string name="preciseLocationEmptyListHint">Inga webbplatser har fått åtkomst till enhetsplats</string>
-    <string name="preciseLocationNoSystemPermission">Aktivera åtkomst till enhetsplats för denna app i Androids Inställningar för att hantera åtkomst till enhetsplats som du har beviljat för enskilda webbplatser.</string>
-    <string name="preciseLocationActivityTitle">Position</string>
+    <string name="settingsPreciseLocation">Plats</string>
+    <string name="preciseLocationFeatureDescription">Webbplatser du besöker kan begära åtkomst till din enhets platsinf\bormation. De får inte åtkomst till din enhets platsinf\bormation förrän du uttryckligen beviljat detta. DuckDuckGo använder enbart din enhets platsinf\bormation anonymt för att tillhandahålla lokala sökresultat.</string>
```
Comment (cmonfortep) [inline app/src/main/res/values-sv/strings.xml:513]: Notice this file is full of non-escaped `\` in all location related strings.

File: app/src/main/res/values-es/strings.xml [hunk 10]
```diff
@@ -395,20 +397,20 @@
...
     <string name="daxDialogGotIt">Entendido</string>
     <string name="hideTipsTitle">¿Ocultar los consejos restantes?</string>
-    <string name="hideTipsText">Sólo hay unos cuantos, y tratamos de hacerlos informativos.</string>
+    <string name="hideTipsText">Solo hay unos cuantos, e intentamos que sean informativos.</string>
     <string name="hideTipsButton">Ocultar consejos para siempre</string>
 
     <!-- New Onboarding Experience -->
     <string name="onboardingWelcomeTitle">"¡Bienvenido a DuckDuckGo!"</string>
-    <string name="onboardingDaxText"><![CDATA[Internet puede ser un poco espeluznante.<br/><br/>¡No te preocupes! Buscar y navegar con privacidad es más fácil de lo que piensas.]]></string>
+    <string name="onboardingDaxText"><![CDATA[Internet puede ser un lugar horrible.\n\nNo te preocupes. Buscar y navegar de forma privada es más fácil de lo que piensas.]]></string>
```
Comment (cmonfortep) [inline app/src/main/res/values-es/strings.xml:410]: `<br/>` replaced by `\n` will not work here.

**Review by cmonfortep — COMMENTED** [review]
Found a few issues in different files:
- `<br/>` replaced by `\n` inside `<CDATA` blocks. New line will not work.
- usage of `\` in xml needs to be `\\`.

File: app/src/main/res/values-hr/strings.xml (hunk 1 above)
Comment (aitorvs) [reply app/src/main/res/values-hr/strings.xml:421]: done

File: app/src/main/res/values-fr/strings.xml (hunk 2 above)
Comment (aitorvs) [reply app/src/main/res/values-fr/strings.xml:410]: done

File: app/src/main/res/values-fr/strings.xml (hunk 3 above)
Comment (aitorvs) [reply app/src/main/res/values-fr/strings.xml:413]: done

File: app/src/main/res/values-lv/strings.xml (hunk 4 above)
Comment (aitorvs) [reply app/src/main/res/values-lv/strings.xml:414]: done

File: app/src/main/res/values-nb/strings.xml (hunk 5 above)
Comment (aitorvs) [reply app/src/main/res/values-nb/strings.xml:423]: done

File: app/src/main/res/values-nl/strings.xml (hunk 6 above)
Comment (aitorvs) [reply app/src/main/res/values-nl/strings.xml:410]: done

File: app/src/main/res/values-sv/strings.xml (hunk 7 above)
Comment (aitorvs) [reply app/src/main/res/values-sv/strings.xml:506]: done

File: app/src/main/res/values-sv/strings.xml (hunk 8 above)
Comment (aitorvs) [reply app/src/main/res/values-sv/strings.xml:501]: done

File: app/src/main/res/values-sv/strings.xml (hunk 9 above)
Comment (aitorvs) [reply app/src/main/res/values-sv/strings.xml:513]: done

File: app/src/main/res/values-es/strings.xml (hunk 10 above)
Comment (aitorvs) [reply app/src/main/res/values-es/strings.xml:410]: done

**Comment by aitorvs** [comment]
> Found a few issues in different files:
> 
> * `<br/>` replaced by `\n` inside `<CDATA` blocks. New line will not work.
> * usage of `\` in xml needs to be `\\`.

@cmonfortep thanks for the review. Addressed them all inhttps://github.com/duckduckgo/Android/pull/1024/commits/b8ac44840114262f1d5badcf336c32fa515aefaf

File: app/src/main/res/values-sv/strings.xml [hunk 11]
```diff
@@ -496,36 +498,36 @@
...
-    <string name="preciseLocationToggleText">Webbplatser kan be om åtkomst till min enhetsplats</string>
-    <string name="preciseLocationEmptyListHint">Inga webbplatser har fått åtkomst till enhetsplats</string>
-    <string name="preciseLocationNoSystemPermission">Aktivera åtkomst till enhetsplats för denna app i Androids Inställningar för att hantera åtkomst till enhetsplats som du har beviljat för enskilda webbplatser.</string>
-    <string name="preciseLocationActivityTitle">Position</string>
+    <string name="settingsPreciseLocation">Plats</string>
+    <string name="preciseLocationFeatureDescription">Webbplatser du besöker kan begära åtkomst till din enhets platsinf\\bormation. De får inte åtkomst till din enhets platsinf\\bormation förrän du uttryckligen beviljat detta. DuckDuckGo använder enbart din enhets platsinf\bormation anonymt för att tillhandahålla lokala sökresultat.</string>
```
Comment (cmonfortep) [inline app/src/main/res/values-sv/strings.xml:513]: Double checking this one: I've noticed that in some parts we are using `platsinformation` and in others `platsinf\bormation`. The latter doesn't exist, however `platsinformation` means "location information" which seems the correct one. Could we use `platsinformation` in all the copies here?

**Review by cmonfortep — APPROVED** [review]
Thanks, please check the comment about the copy used in Swedish, it seems not correct.

Also, I found 4 copies still using `\n` inside a `CDATA` block. 
image

Apart from that, we are good to merge this. thanks!

File: app/src/main/res/values-sv/strings.xml (hunk 11 above)
Comment (aitorvs) [reply app/src/main/res/values-sv/strings.xml:513]: I will change to `platsinformation`

**Comment by aitorvs** [comment]
> Thanks, please check the comment about the copy used in Swedish, it seems not correct.
> 
> Also, I found 4 copies still using `\n` inside a `CDATA` block.
> 
> Apart from that, we are good to merge this. thanks!

@cmonfortep fixed in https://github.com/duckduckgo/Android/pull/1024/commits/7501b20d39864092897a4839652a61fdc4662a98

**Pull request merged**